### ISSUE TITLE
Invariant Violation: Level "fatal" not one of log,info,warn,error,mustfix

### ISSUE THREAD

**Issue by cancan101** [facebook/react-native#4478]
```
Uncaught Error: Invariant Violation: Level "fatal" not one of log,info,warn,error,mustfix
invariant @ invariant.js:39
logIfNoNativeHook @ RCTLog.js:30
messageHandlers.executeJSCall @ debuggerWorker.js:25
onmessage @ debuggerWorker.js:42
```

**Comment by chirag04** [comment]
@cancan101 can you provide more information on this?

**Comment by cancan101** [comment]
I am getting this on RN 0.15 when there is an exception caused in a `Component` constructor.

**Comment by cancan101** [comment]
I also see:
```
[object Object]
handleException @ ExceptionsManager.js:63
handleError @ InitializeJavaScriptAppEngine.js:76
ErrorUtils.reportFatalError @ error-guard.js:28
guard @ MessageQueue.js:41
callFunctionReturnFlushedQueue @ MessageQueue.js:82
messageHandlers.executeJSCall @ debuggerWorker.js:25
onmessage @ debuggerWorker.js:42
```

**Comment by cancan101** [comment]
Looks like this might have been fixed here: https://github.com/facebook/react-native/commit/4b78ed2123e638f4169ee59fede35ac5ec204107

**Comment by ide** [comment]
I remember @javache posting that he fixed a bug that sounds very similar to this one. I'll close this issue but please open a new one if the bug has not been fixed in 0.16.
